source/operations/monitoring/healthcheck-probe.rst

```diff
@@ -16,6 +16,21 @@ endpoints return an HTTP status code indicating whether the underlying
 resource is healthy or satisfies read/write quorum. MinIO exposes no other data
 through these endpoints.
 
+.. cond:: windows
+
+   .. admonition:: Checks Using PowerShell
+      :class: note
+
+      When using PowerShell to run the following the commands, you must explicitly set ``-UserAgent ""``.
+      Otherwise, the healthcheck redirects to the Admin Console instead of the desired Healthcheck API.
```

Comment:
This is not needed anymore @djwfyi MinIO server now handles health requests from user-agents with Mozilla in them.

Reply:
Thanks. Removed the admonition.